alvr/filesystem-layout/src/lib.rs

```diff
@@ -0,0 +1,142 @@
+#[macro_use]
+extern crate lazy_static;
+
+pub struct Layout {
+    // directory to register in openVR driver path
+    pub openvr_driver_dir: std::path::PathBuf,
+    // directory containing default presets
+    pub presets_dir: std::path::PathBuf,
+    // (linux only) path to the executable to wrap vrcompositor
+    pub vrcompositor_wrapper: std::path::PathBuf,
+    // path to the launcher executable
+    pub launcher_exe: std::path::PathBuf,
+    // directory containing dashboard static resources (javascript, css, images)
+    pub dashboard_resources_dir: std::path::PathBuf,
+}
+
+impl Layout {
+    // path to the shared library to be loaded by openVR
+    pub fn openvr_driver_lib(&self) -> std::path::PathBuf {
+        if cfg!(windows) {
+            self.openvr_driver_dir
+                .join("bin/win64/driver_alvr_server.dll")
+        } else {
+            self.openvr_driver_dir
+                .join("bin/linux64/driver_alvr_server.so")
+        }
+    }
+    // path to the manifest file for openVR
+    pub fn openvr_driver_manifest(&self) -> std::path::PathBuf {
+        self.openvr_driver_dir.join("driver.vrdrivermanifest")
+    }
+}
+
+lazy_static! {
+    pub static ref LAYOUT: Layout = if cfg!(windows) {
+        Layout {
+            openvr_driver_dir: std::path::PathBuf::from(""),
```

Comment:
Maybe use an [`Option`](https://doc.rust-lang.org/std/option/index.html)?

Reply:
Not for this one, on windows the path to the manifest is th same as alvr dir. Maybe "." would be more explicit?

Reply:
Oh yeah `.` sounds good.

Reply:
done, and added some comments on the struct